src/main/java/tech/gruppone/stalker/server/model/api/OrganizationDataDto.java

```diff
@@ -20,7 +20,7 @@
   @NonNull @Singular List<PlaceDto> places;
 
   // Using the boxed type because both jackson and lombok behave strangely with primitive booleans
-  @NonNull @Builder.Default Boolean isPrivate = false;
+  @NonNull Long isPrivate;
```

Comment:
come mai questo? non veniva mappato correttamente a bool?

Reply:
Adesso nel db è un tinyint(1), quindi il valore o è 0 o è 1 (mi baso sul create.sql che c'è nel master)

Reply:
ah il tinyint in realtà è il bool di mysql! pensavo r2dbc riuscisse a trasformarlo in boolean. ti dava errore?

Reply:
Eh sì, mi da errore. Ecco perchè l'ho trasformato così!